apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/SpringConfiguration.java

```diff
@@ -363,6 +371,37 @@ public MetricRegistry metricRegistry() {
     final JmxReporter reporter = JmxReporter.forRegistry(metricRegistry).build();
     reporter.start();
 
+    String newRelicMetricKey = System.getenv("NEW_RELIC_METRIC_KEY");
+
+    if (newRelicMetricKey != null) {
+      String newRelicAppName = System.getenv("NEW_RELIC_APP_NAME");
+
+      String hostname;
+      try {
+        hostname = InetAddress.getLocalHost().getHostName();
+      } catch (UnknownHostException e) {
+        hostname = "unknown";
+      }
+
+      SenderConfiguration configuration =
+          SenderConfiguration.builder("https://gov-metric-api.newrelic.com", "/metric/v1")
```

Comment:
Will this never change?  Should this be an Env var as well?

Reply:
Oh, I wouldn't say never. In particular the `v1` to `v2` seems like it has potential to change in the future. But at that point we're likely making code changes anyway.

Reply:
Yes, I would put the host address in an env var.

Reply:
For a few reason. One, we may need this to change depending on the environment in the future. Two, I could see someone, somewhere, considering this semi-sensitive info. We've been pinged for similar things in the past. My two cents FWIW :)

Reply:
I'll change them to environment variables for easy of configuration. But if anybody ever comes in worried about those two values being semi-sensitive, I'd point them to the [public New Relic documentation where they are listed](https://docs.newrelic.com/docs/security/security-privacy/compliance/fedramp-compliant-endpoints/#metric-api).